src/symbol/get_anchors.js

```diff
@@ -8,7 +8,68 @@ import checkMaxAngle from './check_max_angle';
 import type Point from '@mapbox/point-geometry';
 import type {Shaping, PositionedIcon} from './shaping';
 
-export default getAnchors;
+export { getAnchors, getCenterAnchor };
+
+function getLineLength(line: Array<Point>): number {
+    let lineLength = 0;
+    for (let k = 0; k < line.length - 1; k++) {
+        lineLength += line[k].dist(line[k + 1]);
+    }
+    return lineLength;
+}
+
+function getAngleWindowSize(shapedText: ?Shaping,
+                            glyphSize: number,
+                            boxScale: number): number {
+    return shapedText ?
+        3 / 5 * glyphSize * boxScale :
```

Comment:
a comment explaining the magic 3/5 number might be a good idea 🔢

Reply:
@ansis or @nickidlugash do you know the origin of this magic number? I guess I just assumed it was something "experimentally reasonable".